.travis.yml

```diff
@@ -1,6 +1,10 @@
 language: python
 env:
-  - PYTHON_VERSION=2.7
+  global:
+    - PYTHON_VERSION=2.7
+  matrix:
```

Comment:
This looks fine, but can you add  a check for `if test_add_studies` in the `after_success` section so we are not running coveralls twice?